doc/source/new/fqllogging.rst

```diff
@@ -239,74 +235,186 @@ The syntax of the nodetool enablefullquerylog command is as follows:
    -u <username>, --username <username>
   Remote jmx agent username
 
-Run the following command on each node in the cluster.
+To enable FQL, run the following command on each node in the cluster on which you want to enable logging:
 
 ::
 
  nodetool enablefullquerylog --path /tmp/cassandrafullquerylog
 
-After the full query logging has been  enabled run some CQL statements to generate full query logs.
+Disabling or resetting FQL
+-------------
+
+Use the ``nodetool disablefullquerylog`` to disable logging. 
+Use ``nodetool resetfullquerylog`` to stop FQL and clear the log files in the configured directory.
+**IMPORTANT:** Using ``nodetool resetfullquerylog`` will delete the log files! Do not use this command unless you need to delete all log files.
+
+fqltool
+-------
+
+The ``fqltool`` command is used to view (dump), replay, or compare logs.
+``fqltool dump`` converts the binary log files into human-readable format; only the log directory must be supplied as a command-line option.
+
+``fqltool replay`` (`CASSANDRA-14618 <https://issues.apache.org/jira/browse/CASSANDRA-14618>`_) enables replay of logs. 
+The command can run from a different machine or cluster for testing, debugging, or performance benchmarking. 
+The command can also be used to recreate a dropped database object.
+Use ``fqltool replay`` to record and compare different runs of production traffic against different versions/configurations of Cassandra or different clusters.
+Another use is to gather logs from several machines and replay them in “order” by the timestamps recorded.
+
+The syntax of ``fqltool replay`` is:
+
+::
+
+  fqltool replay [--keyspace <keyspace>] [--results <results>]
```

Comment:
@polandll you will likely have merge conflicts somewhere along these lines as this was just merged recently.

https://github.com/apache/cassandra/commit/00c22232d96b6e592fb1d1f32a6b649b49eceb34

Reply:
Thanks for the heads-up. I resolved the conflict.